amd/src/annoto.js

```diff
@@ -427,90 +376,79 @@ define([
-            var annotoIframeClient = "https://cdn.annoto.net/widget-iframe-api/latest/client.js",
+        setupWistiaIframeEmbed: function() {
+            const annotoIframeClient = "https://cdn.annoto.net/widget-iframe-api/latest/client.js",
                 annotoIframeUrl = /https:\/\/fast.wistia.net\/embed\/iframe.*https:\/\/cdn.annoto.net/,
                 wistiaplayers = document.querySelectorAll('iframe');
 
             wistiaplayers.forEach((iframe) => {
-                var iframeSrc = decodeURIComponent(iframe.src);
+                const iframeSrc = decodeURIComponent(iframe.src);
                 if (iframeSrc.match(annotoIframeUrl)) {
                     require([annotoIframeClient], this.setupWistiaIframeEmbedPlugin.bind(this, iframe));
                     return;
                 }
             });
         },
 
-        setupWistiaIframeEmbedPlugin: function(iframe, AnnotoIframeApi){
-            var params = this.params,
+        setupWistiaIframeEmbedPlugin: function(iframe, AnnotoIframeApi) {
+            const params = this.params,
                 annoto = new AnnotoIframeApi.Client(iframe);
 
-            annoto.onSetup(function (next) {
+            annoto.onSetup(function(next) {
                 next({
                     clientId: params.clientId,
-                    position: params.position,
-                    features: {
-                        tabs: params.featureTab,
-                        cta: params.featureCTA,
-                    },
-                    align: {
-                        vertical: params.alignVertical,
-                    },
                     ux: {
                         ssoAuthRequestHandle: function() {
                             window.location.replace(params.loginUrl);
                         },
-                        openOnLoad: params.openOnLoad,
-                        sidePanelLayout: params.sidePanelLayout,
-                        sidePanelFullScreen: params.sidePanelFullScreen,
                     },
-                    widgets: [{
-                        player: {
-                            mediaDetails: function() {
-                                return {
+                    hooks: {
+                        mediaDetails: function() {
+                            return {
+                                details: {
                                     title: params.mediaTitle,
-                                    description: params.mediaDescription,
-                                    group: {
-                                        id: params.mediaGroupId,
-                                        type: 'playlist',
-                                        title: params.mediaGroupTitle,
-                                        privateThread: params.privateThread,
-                                    }
-                                };
-                            },
+                                    id: params.mediaId,
```

Comment:
as in other comment, will break exiting customer's discussions

Reply:
ready for testing